src/shared/components/post/post-listings.tsx

```diff
@@ -113,6 +139,49 @@ export class PostListings extends Component<PostListingsProps, any> {
     );
   }
 
+  handleHighlight(postIndex: number | undefined) {
+    postIndex != undefined && this.setState({ highlightedPost: postIndex });
+  }
+
+  toggleExpand() {
+    this.setState({ isExpanded: !this.state.isExpanded });
+  }
+
+  handleKeybinds(i: PostListings, event: KeyboardEvent) {
+    const idx = i.state.highlightedPost;
+    if (
+      !i.props.viewOnly &&
+      !event.ctrlKey &&
+      !event.metaKey &&
+      !event.altKey
```

Comment:
If you want to avoid needing a not in front of everything, you could apply [DeMorgan's law](https://betterprogramming.pub/de-morgans-laws-in-software-engineering-b08feb897f75).